cmd/cachecmd/main.go

```diff
@@ -327,11 +341,16 @@ func cacheDir() string {
 
 // REF: https://specifications.freedesktop.org/basedir-spec/basedir-spec-0.6.html
 func xdgCacheHome() string {
-	path := os.Getenv("XDG_CACHE_HOME")
-	if path == "" {
-		path = filepath.Join(os.Getenv("HOME"), ".cache")
+	home := ""
+	if runtime.GOOS == "windows" {
+		home = os.Getenv("USERPROFILE")
```

Comment:
if windows user set XDG_CACHE_HOME, maybe we should prefer XDG_CACHE_HOME. WDYT?